x-pack/plugins/ml/server/saved_objects/initialization/initialization.ts

```diff
@@ -106,10 +106,22 @@ export function jobSavedObjectsInitializationFactory(
     //   index: '.ml-config',
     // });
     // return body.count > 0;
+    let adJobsCount = 0;
+    let dfaJobsCount = 0;
 
-    const adJobs = await client.asInternalUser.ml.getJobs();
-    const dfaJobs = await client.asInternalUser.ml.getDataFrameAnalytics();
-    return adJobs.count > 0 || dfaJobs.count > 0;
+    try {
+      const adJobs = await client.asInternalUser.ml.getJobs();
+      adJobsCount = adJobs.count;
+    } catch (error) {
+      // ignore errors as anomaly detection may not be enabled
```

Comment:
is it worth adding a logger output of the `debug` or `warning` level?

Reply:
it's probably not worth it. this code won't be around for long, catching these errors just allows kibana to start up cleanly for now until we add the correct checks for what features are enabled in the serverless environment.

Reply:
Actually, I've changed my mind. There's no harm in logging them to the debug level for now. Just in case.

Reply:
Updated in https://github.com/elastic/kibana/pull/158749/commits/b2949e7c8976ef02e72bdba221f72011b7540496